redshift_sqlalchemy/dialect.py

```diff
@@ -72,6 +82,7 @@ def post_create_table(self, table):
         text = ""
         info = table.dialect_options['redshift']
         diststyle = info.get('diststyle', None)
```

Comment:
not changed, but `info.get('diststyle', None)` is equivalent to `info.get('diststyle')`